partner_compassion/models/partner_compassion.py

```diff
@@ -79,12 +79,14 @@ def _get_receipt_types(self):
     partner_duplicate_ids = fields.Many2many(
         'res.partner', 'res_partner_duplicates', 'partner_id',
         'duplicate_id', readonly=True)
+    church_member_count = fields.Integer(compute='_is_church',
+                                         readonly=True)
 
     ##########################################################################
     #                             FIELDS METHODS                             #
     ##########################################################################
     @api.multi
-    @api.depends('category_id')
+    @api.depends('category_id', 'member_ids', 'church_member_count')
```

Comment:
Remove `church_member_count`. You cannot make your computed field depend on itself, it will produce bad behaviour.